README.md

```diff
@@ -201,6 +201,7 @@ Name | Website | Region
 [Evrone](/company-profiles/evrone.md) | https://evrone.com/ | Worldwide
 [ExportData](/company-profiles/exportdata.md) | https://www.exportdata.io/ | Worldwide
 [Eyeo (Adblock Plus)](/company-profiles/eyeo.md) | https://eyeo.com/ | Worldwide
+[FactorialHR](/company-profiles/factorialhr.md) | https://www.factorialhr.com | Europe
```

Comment:
The region here is where you hire, so should this be worldwide?

Reply:
Fixed, thanks for the comments 👏